tests/integration/targets/azure_rm_networkinterface/tasks/main.yml

```
@@ -10,31 +10,47 @@
   azure_rm_virtualnetwork:
       resource_group: "{{ resource_group_secondary }}"
       name: "tn{{ rpfx }}"
-      address_prefixes: "10.10.0.0/16"
+      address_prefixes: ["10.10.0.0/16","fdae:f296:2787::/48"]
   register: vn
 
 - name: Add subnet
   azure_rm_subnet:
       resource_group: "{{ resource_group_secondary }}"
       name: "tn{{ rpfx }}"
-      address_prefix: "10.10.0.0/24"
+      address_prefixes_cidr: ["10.10.0.0/24","fdae:f296:2787::/64"]
```

too few spaces after comma
```suggestion
      address_prefixes_cidr: ["10.10.0.0/24", "fdae:f296:2787::/64"]
```